### DISCUSSION TITLE
Compile error: Could not write lines to file "Localization/.placeholder".

### DISCUSSION THREAD

**Discussion by orchardc** [OrchardCMS/OrchardCore#15897] [Q&A]
In the new release (72282) , when compile I got the following error: Could not write lines to file "Localization/.placeholder".

**Comment by deanmarcussen** [comment]
Can you provide some details on the environment you are running?

Operating System etc?

**Comment by jtkech** [comment]
In addition to the `wwwroot` folder a cms app now needs a `Localization` folder whose files are outputed on publishing. See this folder (which includes a `.placeholder` file) under our `OC.Cms.Web`app.

So, if you build your own cms app you need to physically create this folder, then a placeholder file is automatically created on building if it is not already there.

In your project file you can use the following if you want to show this folder under the solution.

    <ItemGroup>
      <Folder Include="wwwroot\" />
      <Folder Include="Localization\" />
    </ItemGroup>

**Comment by orchardc** [comment]
Thanks @jtkech, it works !
